dataplane/pkg/tests/memif_proxy_test.go

```diff
@@ -0,0 +1,18 @@
+package tests
```

Comment:
Not sure that this test will be running on CI. Will check tomorrow

Reply:
Please rename the file to start with `basic_` and add `// +build basic` on top. Consult the other tests in the folder.

Reply:
@nickolaev as I know the basic prefix using for integrations tests. This is a simple unit test. Should this test have a prefix?

Reply:
RIght, scratch my comment. It's my confusion.